CHANGELOG.md

```diff
@@ -66,6 +66,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - [#7182](https://github.com/apache/trafficcontrol/pull/7182) *Traffic Control Cache Config (t3c)* Sort peers used in strategy.yaml to prevent false positive for reload.
 - [#7204](https://github.com/apache/trafficcontrol/pull/7204) *Traffic Control Cache Config (t3c)* strategies.yaml hash_key only for consistent_hash
 
+### Removed
+- Remove components in `infrastructre/docker/`, not in use, cdn-in-a-box performs the same functionality
+- Remove `misc/jira_github_issue_import.py`, the project does not use JIRA
+- Remove `traffic_ops/install/bin/_postinstall.pl` and `traffic_ops/install/lib/`, _postinstall.py is the recommended script to use for install
+- Remove `traffic_ops/install/bin/convert_profile/`, this script is outdated and is for use on an EOL ATS version
+- Remove `traffic_ops/install/bin/install_go.sh`, `traffic_ops/install/bin/todb_bootstrap.sh` and `traffic_ops/install/bin/install_goose.sh` are no longer in use
+- 
+
```

Comment:
These need a PR or issue link, and you've got an extra blank list item at the end.